packages/http/lib/auth.tsp

```diff
@@ -27,7 +31,7 @@ enum AuthType {
  * ```
  */
 @doc("")
-model BasicAuth {
+model BasicAuth extends SecurityScheme {
```

Comment:
I think this and the constrain change might make it impossible to define a custom scheme